paas/install_client.go

```diff
@@ -27,13 +29,40 @@ const (
 // installing Carrier on it.
 type InstallClient struct {
 	kubeClient *kubernetes.Cluster
+	options    *kubernetes.InstallationOptions
 	ui         *ui.UI
 	config     *config.Config
 	Log        logr.Logger
 }
 
+func NewInstallClient(flags *pflag.FlagSet, options *kubernetes.InstallationOptions) (*InstallClient, func(), error) {
```

Comment:
same here but with the addition of the `options` parameter. This parameter was the main reason for the removal of wire in the PR. Maybe there are workarounds but they certainly look uglier than removing wire.